nix/workbench/shell.nix

```diff
@@ -88,7 +88,6 @@ in project.shellFor {
   nativeBuildInputs = with pkgs; with haskellPackages; with cardanoNodePackages; [
     cardano-ping
     cabalWrapped
-    db-analyser
```

Comment:
@newhoggy, can we avoid removing `db-analyser` -- since that breaks the workbench?